### ISSUE TITLE
Table Style changed

### ISSUE THREAD

**Issue by trampas** [python-openxml/python-docx#164]
it appears that the default table style has change, specifically the code on front page does not create tables that look like example.

**Comment by scanny** [comment]
Yes, this had to go when we added styles support in release 0.8.0. Trying to be "helpful" by picking a table style caused problems when folks used a starting document that didn't have that style defined. You can achieve the same result now with this:
```python
document.add_table(rows, cols, 'Light Shading Accent 1')
```

Note that if that style is not defined in your document, python-docx will raise an exception.

I'll leave this open to update the front-page example.

**Comment by trampas** [comment]
Thanks!

Also on the style doing table.style='Light Shading Accent 1' will create a
depreciated warning, thus the documents might need updated to indicate the
new way...

**Comment by scanny** [comment]
I can't reproduce that behavior @trampas.

Although I do get a deprecation warning for this:
```python
table = document.add_table(2, 2, 'LightShading-Accent1')
```

The former is the table style *name*, where the latter is the table style *id*. The deprecation warning is triggered by using the style id, which will not be supported going forward.
